data-products/src/main/scala/org/sunbird/analytics/job/report/AssessmentMetricsJob.scala

```diff
@@ -75,58 +75,50 @@ object AssessmentMetricsJob extends optional.Application with IJob with BaseRepo
     * @param settings - Cassnadra configs
     * @return
     */
-  def loadData(spark: SparkSession, settings: Map[String, String]): DataFrame = {
+  def loadData(spark: SparkSession, settings: Map[String, String], url: String): DataFrame = {
```

Comment:
The loadData function now seems to handle loading data from Redis as well. But, the function comment says otherwise.

Reply:
Updated comments